src/components/CharacterLimitMainText/CharacterLimit.spec.tsx

```diff
@@ -0,0 +1,46 @@
+import { render, screen } from '@testing-library/react';
+
+import CharacterLimit from './CharacterLimit.tsx';
+
+describe('CharacterLimit Component', () => {
+  it('render the component without svg', () => {
+    render(
+      <CharacterLimit id={'test'} maxLength={10} svg={null} value={'1234'} />
```

Comment:
```suggestion
      <CharacterLimit id='test' maxLength={10} svg={null} value='1234' />
```

String nao precisa ficar entre brackets.